features/nanostack/mbed-mesh-api/source/NanostackEthernetInterface.cpp

```diff
@@ -97,5 +97,6 @@ nsapi_error_t NanostackEthernetInterface::do_initialize()
 
 nsapi_error_t Nanostack::EthernetInterface::bringdown()
 {
-    return enet_tasklet_disconnect(true);
+    (void)enet_tasklet_disconnect(true);
```

Comment:
This should be turning an error into some sort of nsapi error code, rather than swallowing it. I guess DEVICE_ERROR would do.

Reply:
I did contemplate that (it does make sense), but the original code swallowed it, and without a way to test it, I didn't think it would be good to alter the behavior.